src/main/java/org/codehaus/mojo/license/header/transformer/PropertiesFileHeaderTransformer.java

```diff
@@ -67,13 +67,13 @@ public String addHeader( String header, String content )
 
             // shebang was detected
 
-            int endProlog = content.indexOf( getLineSeparator() );
+            int endProlog = content.indexOf( '\n', startProlog );
```

Comment:
I think the fix is wrongly assuming that all possible shell scripts are delimited by `\n`. I vote for being as defensive as possible here - i.e. we should accept both an `\r\n` pair or `\n` which ever of them is present. Something like you did here: https://github.com/mojohaus/license-maven-plugin/pull/189/files#diff-0dbbbe109835879abe30a4c74cf1e191R131 WDYT?

Reply:
The fix considers both delimiters (`\r\n` and `\n`) : the variable endProlog will point to the end of the line no matter what convention is used.

I tested both cases in my machine, but couldn't commit the integration test with the `\r\n` scenario because the editorconfig verification failed (it is expecting all the plugin code, including tests, to have  LF end of lines)

https://github.com/mojohaus/license-maven-plugin/blob/511684ec861fa0b84f5075527154c50657b99b99/.editorconfig#L8 

and I didn't think was a good idea to modify the main POM just to make an exception for this integration test

Reply:
> couldn't commit the integration test with the `\r\n` scenario because the editorconfig verification failed (it is expecting all the plugin code, including tests, to have LF end of lines)

Just add a new rule at the end of the top level `.editorconfig` file, something like

```
[src/it/my-test/my-file.ext]
end_of_line = crlf 
```

or you could even create a new `.editorconfig` file in the root folder of your test and add an appropriate rule there.

Reply:
> The fix considers both delimiters (`\r\n` and `\n`) : the variable endProlog will point to the end of the line no matter what convention is used.

Indeed, thanks for correcting me.